### PR TITLE
chore: remove last afgo and orb dependencies

### PR DESCRIPTION
**Pull request by Moopli**

### CODE REVIEW

**Review by rolsonquadras — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -180,13 +180,7 @@ clean:
 
 .PHONY: update-aries
 update-aries:
```
Comment (DRK3) [inline Makefile:182]: Can this target just be removed completely?

File: component/wallet-cli/internal/storage/leveldb/leveldb.go [hunk 2]
```diff
@@ -0,0 +1,661 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
```
Comment (DRK3) [inline component/wallet-cli/internal/storage/leveldb/leveldb.go:2]: Copyrights should be updated

File: Makefile (hunk 1 above)
Comment (Moopli) [reply Makefile:182]: Not sure if it's used elsewhere, so I've left it with a deprecation notice for now.

File: component/wallet-cli/internal/storage/leveldb/leveldb.go (hunk 2 above)
Comment (Moopli) [reply component/wallet-cli/internal/storage/leveldb/leveldb.go]: Done.

**Review by DRK3 — APPROVED** [review]

**Pull request merged**